.kiro/steering/tech.md

```
@@ -0,0 +1,80 @@
+# QuickLog-Solo: 技術スタック・開発環境
+
+## 言語・技術
+| 分類 | 技術 |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)